### ISSUE TITLE
9.4.1.2/3.8 In the "Details" section under the "Activity" tab, it is possible to open a dropdown list when entering "@", ":" or "/" into the textbox, for example, to select from a list of emojis. The elements in these lists are not accessible to screen reader users.

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42231]
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v3-n8

9.4.1.2/3.8 In the "Details" section under the "Activity" tab, it is possible to open a dropdown list when entering "@", ":" or "/" into the textbox, for example, to select from a list of emojis. The elements in these lists are not accessible to screen reader users.

**Comment by JuliaKirschenheuter** [comment]
duplicate of https://github.com/nextcloud/server/issues/41896
